docker-storage-setup.conf

```diff
@@ -1,20 +1,21 @@
-# Specify storage driver one wants to use with docker. Default is devicemapper.
+# Specify storage driver one wants to use with container runtimes.
+# Default is devicemapper.
 # Other possible options are overlay, overlay2 and "". Empty string means do
 # not do any storage setup.
 STORAGE_DRIVER=devicemapper
 
-# A set of extra options that should be appended to the generated
-# DOCKER_STORAGE_OPTIONS string. These options will be passed to the Docker
-# daemon as-is and should be valid Docker storage options.
-# EXTRA_DOCKER_STORAGE_OPTIONS="--storage-opt dm.fs=ext4"
+# Set extra options that will be appended to the generated STORAGE_OPTIONS
```

Comment:
This will continue to be DOCKER_STORAGE_OPTIONS for a while till we figure out a way how to make output string generic as well as useful. May be user will have to pass in a prefix so that we could have DOCKER_STORAGE_OPTIONS in output?

Or always output STORAGE_OPTIONS and modify docker to accept that.

Anyway, this should be modified when associated code has been modified too. Possibly a separate patch for this.